registry/migrations/20230828111733_users.up.sql

```diff
@@ -1,37 +1,36 @@
 -- table of users.
-CREATE TABLE "users" (
-    "id"          SERIAL PRIMARY KEY,
-    "handle"      TEXT NOT NULL UNIQUE,
-    "created_at"  TIMESTAMPTZ NOT NULL DEFAULT (now()),
-    "updated_at"  TIMESTAMPTZ NOT NULL DEFAULT (now()),
+CREATE TABLE users (
```

Comment:
I meant that more as a general principle. The idea of having migrations in the first place is that it's an append-only log. If we're in a stage where we can modify existing migrations, we could as well squash them and operate on just one *.sql file.

Reply:
You should squash all migrations introduced during the development cycle of a PR, or otherwise you accumulate tons of unnecessary migration steps.

I don't know about sqlx, but usually the definitive way this works even without special tooling support is by replaying the migrations to the state before the MR and then capture the diff for the target state at the "end" of the PR.